ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/425__model_serving_ovms_on_kserve.robot

```diff
@@ -0,0 +1,207 @@
+Resource          ../../../Resources/OCP.resource
+Resource          ../../../Resources/CLI/ModelServing/modelmesh.resource
+Suite Setup       Model Serving Suite Setup
+Suite Teardown    Model Serving Suite Teardown
+Test Tags         ModelMesh
+
+*** Variables ***
+${INFERENCE_INPUT}=    @ods_ci/tests/Resources/Files/modelmesh-mnist-input.json
+${INFERENCE_INPUT_OPENVINO}=    @ods_ci/tests/Resources/Files/openvino-example-input.json
+${PRJ_TITLE}=    model-serving-project
+${PRJ_DESCRIPTION}=    project used for model serving tests
+${MODEL_CREATED}=    ${FALSE}
+${MODEL_NAME}=    test-model
+${RUNTIME_NAME}=    Model Serving Test
+${SECOND_PROJECT}=    sec-model-serving-project
+${SECURED_MODEL}=    test-model-secured
+${SECURED_RUNTIME}=    Model Serving With Authentication
+${EXPECTED_INFERENCE_SECURED_OUTPUT}=    {"model_name":"${SECURED_MODEL}__isvc-83d6fab7bd","model_version":"1","outputs":[{"name":"Plus214_Output_0","datatype":"FP32","shape":[1,10],"data":[-8.233053,-7.7497034,-3.4236815,12.3630295,-12.079103,17.266596,-10.570976,0.7130762,3.321715,1.3621228]}]}
+${EXPECTED_INFERENCE_OUTPUT_OPENVINO}=    {"model_name":"${MODEL_NAME}__isvc-8655dc7979","model_version":"1","outputs":[{"name":"Func/StatefulPartitionedCall/output/_13:0","datatype":"FP32","shape":[1,1],"data":[0.99999994]}]}
+
+
+*** Test Cases ***
+# Verify Model Serving Installation
+#     [Documentation]    Verifies that the core components of model serving have been
+#     ...    deployed in the ${APPLICATIONS_NAMESPACE} namespace
+#     [Tags]    Smoke
+#     ...       Tier1
+#     ...       OpenDataHub
+#     ...       ODS-1919
+#     Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify odh-model-controller Deployment
+#     Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify ModelMesh Deployment
+#     Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify Etcd Pod
+
+Verify Openvino_IR Model Via UI
+    [Documentation]    Test the deployment of an openvino_ir model
+    [Tags]    Smoke
+    ...    ODS-2054
+    Create Openvino Models For Kserve    server_name=${RUNTIME_NAME}    model_name=${MODEL_NAME}
+    ...    project_name=${PRJ_TITLE}   num_projects=1
+    [Teardown]    Run Keyword If Test Failed    Get Modelmesh Events And Logs
+    ...    server_name=${RUNTIME_NAME}    project_title=${PRJ_TITLE}
+
+Test Inference Without Token Authentication
+    [Documentation]    Test the inference result after having deployed a model that doesn't require Token Authentication
+    [Tags]    Smoke
+    ...    ODS-2053
+    Run Keyword And Continue On Failure    Verify Model Inference    ${MODEL_NAME}    ${INFERENCE_INPUT_OPENVINO}
+    ...    ${EXPECTED_INFERENCE_OUTPUT_OPENVINO}    token_auth=${FALSE}
+    [Teardown]    Run Keyword If Test Failed    Get Modelmesh Events And Logs
+    ...    server_name=${RUNTIME_NAME}    project_title=${PRJ_TITLE}
```

Comment:
what about moving the inference inside the prev test?

Reply:
Done